lib/migrator/migrationSteps.js

```diff
@@ -66,11 +67,11 @@ module.exports.migrateConfig = async (oldConfigPath) => {
 
     await fse.writeJsonSync(newConfigPath, newConfig, { encoding: 'utf8' })
     await deleteFile(oldConfigPath)
+    result = true
+  } catch (e) {}
```

Comment:
@lfdebrux
***********************************************************************************
I'm currently locked out of gmail and therefore slack and google-meet.  Please if you get this message, let Ruth know so that I can get access again.  I think that whoever is in charge of gmail accounts didn't get the memo that my contract was renewed!  Ruth can contact me at bensurgison@gmail.com

Reply:
logging to the migrate.log as requested

Reply:
Done